dask/array/routines.py

```diff
@@ -1202,7 +1203,7 @@ def piecewise(x, condlist, funclist, *args, **kw):
     )
 
 
-@wraps(chunk.coarsen)
+@wraps(chunk.coarsen)  # this is not a numpy function
```

Comment:
This comment may not make as much sense outside of th context of this PR

Reply:
I meant to write a comment into the PR, that there are a couple of annotations in the diff, meaning they can be discussed here and removed when resolved. I guess I've done that now :)